xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/RegistrationConfig.xml

```diff
@@ -522,9 +522,30 @@
       <passwordRuleOneUpperCaseEnabled>0</passwordRuleOneUpperCaseEnabled>
     </property>
     <property>
-      <registrationSuccessMessage>#set($message = $services.localization.render('core.register.successful', 'xwiki/2.1', ['USERLINK', $userName]))
+      <registrationSuccessMessage>#set($discard = $xwiki.ssx.use("XWiki.RegistrationConfig"))
+#set($displayName = "$!firstName $!lastName")
+#set($noReadableName = ($!firstName == "") || ($!lastName == ""))
+#if($noReadableName)
+  #set($displayName = $userName)
+#end
+#set($headline = $services.localization.render('core.register.successful.welcome', [$displayName]))
 #set($userLink = $xwiki.getUserName("$userSpace$userName"))
-{{info}}$message.replace('USERLINK', "{{html clean=false}}$userLink{{/html}}"){{/info}}</registrationSuccessMessage>
+#set($successAndLogin = $services.localization.render('core.register.successful.successandlogin'))
+[[image:undraw_done_re_oak4.svg||data-xwiki-image-style-alignment="center" height="50vh"]]
```

Comment:
Any reason for not using directly some html to put in the html macro for that one?

Reply:
I'm not sure I understand your point.
I understood that you were asking why I used the wiki syntax for the picture and html only for the content after.

Beforehand, it was only wiki syntax. However I had more advanced needs for styling, and we should avoid inline styling, so I used HTML with classes (in hindsight, I probably still could have kept some wiki syntax here ^^' ). The picture did not have inline style and was at the start so I figured I could just use wiki syntax.